### ISSUE TITLE
Feature request: Cross platform GUI ? - possible contribution

### ISSUE THREAD

**Issue by maximilien-noal** [paulirwin/JavaToCSharp#48]
I can replace the WPF UI with AvaloniaUI with one hand and half asleep in one day.

AvaloniaUI runs on Windows/Mac/Linux/iOS/Android/WASM.

Would you accept such a PR ?

**Comment by paulirwin** [comment]
I would love such a PR! IKVM.NET is adding .NET 6 and arm64 support very soon it seems, so I hope to resume work on getting the core of the app up to date once that is complete. If there is a way to create the UI in the meantime without that support completed, that would be awesome and I would welcome that. Otherwise it might have to wait.

**Comment by paulirwin** [comment]
Although you can't currently build on macOS arm64, .NET 6 support has just been merged into master. This should be a good basis for an AvaloniaUI if you're interested in doing that. Thanks!

**Comment by maximilien-noal** [comment]
Will do !

**Comment by maximilien-noal** [comment]
I'm currently doing it on Windows.

On Linux, the compiler gives an error related to IKVM.

One limitation of the gui is the custom window chrome. The 'fork me on github button' will be part of the window content.

A custom window chrome is supported by AvaloniaUI, but it's not the safest option on Linux with gnome's Client Slide Decorations and other weird window managers.

**Comment by paulirwin** [comment]
I'm fine with moving the GitHub link to an overflow menu or something if that helps.

I haven't tried the build on Linux. What error are you getting?

**Comment by maximilien-noal** [comment]
> I haven't tried the build on Linux. What error are you getting?

Here is the issue:
#52
